quickstart-docker/resources/sample_message.txt

```diff
@@ -0,0 +1,14 @@
+service_type: replace_service_type_here
+order_number: "tolol"
```

Comment:
replace this word "tolol" please , it means "stupid" in bahasa Indonesia language

Reply:
fixed